recipes-devtools/mraa/mraa.inc

```diff
@@ -9,7 +9,7 @@ S = "${WORKDIR}/git"
 
 inherit distutils-base pkgconfig python-dir cmake
 
-EXTRA_OECMAKE_append = "-DINSTALLTOOLS=ON -DFIRMATA=ON -DPYTHON_SITE_DIR:FILEPATH=${PYTHON_SITEPACKAGES_DIR} -DBASE_LIB_INSTALL_DIR=${base_libdir} -DCMAKE_SKIP_RPATH=ON"
+EXTRA_OECMAKE_append = "-DINSTALLTOOLS=ON -DFIRMATA=ON -DPYTHON2_PACKAGES_PATH:FILEPATH=../${PYTHON_SITEPACKAGES_DIR} -DBASE_LIB_INSTALL_DIR=${base_libdir} -DCMAKE_SKIP_RPATH=ON"
```

Comment:
PYTHON_SITEPACKAGES_DIR = "${libdir}/${PYTHON_DIR}/site-packages" - are you sure you want to to go up one level at the beginning and not at the end?

From the description it is not clear to me what "the prefix" is, so your fix might be correct. I was just wondering.